CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Fixes
 
 - Captured [Http Client Errors](https://docs.sentry.io/platforms/dotnet/guides/aspnet/configuration/http-client-errors/) on .NET 5+ now include a full stack trace in order to improve Issue grouping ([#4724](https://github.com/getsentry/sentry-dotnet/pull/4724))
+- Captured [GraphQL Client Errors](https://docs.sentry.io/platforms/dotnet/configuration/graphql-client-errors/) on .NET 5+ now include a full stack trace in order to improve Issue grouping ([#4762](https://github.com/getsentry/sentry-dotnet/pull/4762))
```

Comment:
note: I'm afraid ... as usual (😟) ... please update from `main` and move into the "new" `Unreleased` section, now after [6.0.0-rc.2](https://github.com/getsentry/sentry-dotnet/releases/tag/6.0.0-rc.2) release